### PR TITLE
Add reply-squelch option to notify and unnotify commands.

### PR DESCRIPTION
**Pull request by BrockA**
Warning: not tested.

### CODE REVIEW

File: chatcommunicate.py [hunk 1]
```diff
@@ -424,14 +424,16 @@ def handle_commands(content_lower, message_parts, ev_room, ev_user_id, ev_user_n
             return "2 arguments expected"
         user_id = int(ev_user_id)
         chat_site = wrap2.host
+        # Room id is already known via the ev_room parameter.  Why make the user specify it?
```
Comment (ProgramFOX) [inline chatcommunicate.py]: Because Smokey posts in more rooms than it listens in.

**Comment by ProgramFOX** [comment]
PR looks good to me, but please drop the comments you added (see my line note). Thanks! :)

**Comment by BrockA** [comment]
@ProgramFOX Okay, removed the comments.

**Comment by ProgramFOX** [comment]
Great, thanks!

**Pull request merged**